core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -436,6 +436,23 @@ protected function doReplace($strBuffer, $blnCache)
 							break;
 						}
 
+						$strName = $objNextPage->title;
+						$strTarget = $objNextPage->target ? ' target="_blank" rel="noreferrer noopener"' : '';
+						$strClass = $objNextPage->cssClass ? sprintf(' class="%s"', $objNextPage->cssClass) : '';
+						$strTitle = $objNextPage->pageTitle ?: $objNextPage->title;
+
+						// Early return for tags that do not need the URL
+						switch (strtolower($elements[0]))
+						{
+							case 'link_title':
+								$arrCache[$strTag] = StringUtil::specialcharsAttribute($strTitle);
+								break 2;
+
+							case 'link_name':
+								$arrCache[$strTag] = StringUtil::specialcharsAttribute($strName);
+								break 2;
+						}
```

Comment:
```suggestion
						if ($elements[0] == 'link_title')
						{
							$arrCache[$strTag] = StringUtil::specialcharsAttribute($strTitle);
							break 2;
						}

						if ($elements[0] == 'link_name')
						{
							$arrCache[$strTag] = StringUtil::specialcharsAttribute($strName);
							break 2;
						}
```

Reply:
you know this is a 1:1 copy of the lines below, right? Including the `lowercase` and switch-case?

Reply:
The lines below are no early returns, so you cannot compare the two cases.

Are you aware that you have used complexity as an argument against @ausi's suggestion above while at the same time you refuse to reduce the complexity here? That does not make too much sense IMHO. 🤷‍♂️